package.json

```diff
@@ -32,6 +32,8 @@
     "@typescript-eslint/parser": "^4.29.3",
     "eslint-plugin-jest": "^24.4.0",
     "eslint-plugin-jsdoc": "^36.0.8",
-    "eslint-plugin-prefer-arrow": "^1.2.3"
+    "eslint-plugin-lodash": "^7.3.0",
+    "eslint-plugin-prefer-arrow": "^1.2.3",
+    "eslint-plugin-security": "^1.4.0"
```

Comment:
It looks like this repo hasn't been updated in 3 years, and seems like it isn't being maintained. Maybe we should avoid using it for that reason?

Curious what you think. FWIW, it seems like CodeQL would detect at least some of the problems that this config claims to detect.

Reply:
It does seem the rules are not very solid, and old.  Not sure if it needs to be kept up to date very much.